### ISSUE TITLE
infinite loading-screen / crash on startup on emulator

### ISSUE THREAD

**Issue by Jonas-P282** [roccodev/xc3-file-loader#4]
I have tried getting the file loader to work on both Yuzu and Ryujinx and neither can start xc3 with the mod active. I followed the instructions on the page and  nothing seemed to work. For Yuzu, I even tried deleting the exef folder, since I didn't know what caused the crash. This did allow the game to start and run but no changes from any mods I added afterwards seemed to show in game. (The crashes happen without any other mods as well, so it has something to do with the file loader)

Yuzu has an infinite loading screen and Ryujinx just crashes completely. 

Is this a known issue? I couldn't find anything, so I assume I am doing something very wrong.
I even tried installing it as both normal mods and athmosphere mods on both emulators, with identical results.

What else could I try?
Thanks in advance

**Comment by roccodev** [comment]
Hey! What version of the game are you running? Do you have any DLCs loaded? Also, you mentioned having no other mods; are you using the file loader to replace files (if so, what files?), or does the issue also occur with the file loader and no other files in the directory?

**Comment by Jonas-P282** [comment]
I am using some version after the dlc with all dlc loaded. (I'm currently
away from home and cannot check the ecxact version) The issue happens with
the file loader as the only mod in the folder as well as with any mod  that
requires it as long as the loader ist active. So If the loader ist active,
the game won't load. Deactivating the dlc doesn't seem to change anything
from what i could tell.

Rocco ***@***.***> schrieb am Fr., 9. Aug. 2024, 22:44:

**Comment by Jonas-P282** [comment]
So the version I use is 2.1.1 with all dlc and I have tried to get the game to run with only the file loader as either a regular mod in the mod directory or as an athmosphere mod (with the respective file structure of course) and I have also tried multiple settings with both Vulcan and Open GL, none of which changed anything. I have a log from ryujinx, right before it crashes. Hopefully this helps. 

[Ryujinx_1.1.1401_2024-09-28_15-30-31.log](https://github.com/user-attachments/files/17174582/Ryujinx_1.1.1401_2024-09-28_15-30-31.log)
